measure/endurance-snapshot

```diff
@@ -475,8 +467,14 @@ fi
 echo "- Disk usage"
 LC_ALL=C df -k -P > $path/df
 
-echo "- statefs"
-dump_statefs_value namespaces/Screen/Blanked
+echo "- Blanked state"
+blanked_state=$(qdbus --system com.nokia.mce /com/nokia/mce/request com.nokia.mce.request.get_display_status)
+if [ "$blanked_state" == "on" ]; then
```

Comment:
Depending on what the meaning of blanked state is supposed to be ...

DIsplay is powered on in "dimmed" state too, so using: state != "off" might be more appropriate.
Or reverse the 'if' and test: if state == "off" blanked=1 else blanked = 0

Reply:
This code removed in the mce version.